en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md

```diff
@@ -0,0 +1,139 @@
+# Adding Multiple Keys to the Primary Keystore
+
+WSO2 Identity Server enables adding multiple keys to the primary keystore. Let's explore the following topics to learn more.
+
+!!! tip "Before you begin"
+
+	Sign in to the WSO2 Identity Server Management Console that will be hereafter called Management Console.
+
```

Comment:
Remove this section and link the 'Management Console' phrase to the respective 'Management Console' section.

Reply:
Fixed